admin/services/data-access/group.data.service.js

```diff
@@ -309,11 +309,29 @@ groupDataService.sqlMarkGroupAsDeleted = async (groupId) => {
  */
 groupDataService.sqlFindGroupsByIds = async (schoolId, groupIds) => {
   if (!schoolId || !groupIds || groupIds.length < 3) return false
-  const sql = `SELECT id, name 
+
+  let sql = `SELECT id, name 
     FROM ${sqlService.adminSchema}.[group] 
-    WHERE school_id=${schoolId} 
-    AND id IN (${groupIds})`
+    WHERE school_id=${schoolId}`
+
+  let whereClause = ' AND id IN ('
   const params = []
+  const ids = Object.values(groupIds)
+  for (let index = 0; index < ids.length; index++) {
+    if (ids[index].group_id) {
+      whereClause = whereClause + `@p${index}`
+      if (index < ids.length - 1) {
+        whereClause += ','
+      }
+      params.push({
+        name: `p${index}`,
+        value: ids[index].group_id,
+        type: TYPES.Int
+      })
+    }
+  }
+  whereClause = whereClause + ')'
+  sql = sql + whereClause
```

Comment:
This isn't good practise.  Better to create a new string than modify an existing one.  Most other data services avoid this for the most part.

Reply:
@jon-shipley @GuyHarwood asked me to use 'sqlFindCheckWindowsAssignedToForms' as a reference to write this method, and so I did.